libraries/AC_WPNav/AC_WPNav.h

```diff
@@ -259,17 +259,13 @@ class AC_WPNav
     Vector3f    _destination;           // target destination in cm from ekf origin
     float       _track_error_xy;        // horizontal error of the actual position vs the desired position
     float       _track_scalar_dt;       // time compression multiplier to slow the progress along the track
+    float       _terain_vel;
```

Comment:
Let's put some comments to the right of these member declarations so people know what they're used for.

Reply:
We've replaced _accel_terrain_offset, _vel_terrain_offset, _pos_terrain_offset with _terain_accel and _terain_vel but are we missing the initialisation of these new members?